dali/operators/reader/loader/video_loader.h

```diff
@@ -73,7 +74,7 @@ std::vector<dali::file_meta> get_file_label_pair(const std::string& path,
 }  // namespace filesystem
 
 struct OpenFile {
```

Comment:
I'd rename this to `VideoFile` or similar, since it no longer represents an open file.

Reply:
Done